stdlib/digest/0/digest.rbs

```diff
@@ -398,3 +398,37 @@ end
 
 class Digest::SHA512 < Digest::Base
 end
+
+# https://github.com/seattlerb/minitest/blob/13c48a03d84a2a87855a4de0c959f968001
```

Comment:
Could you delete this comments for `Object`? I'm not sure where it is from. (Minitest?)

Reply:
Sure thing, I removed the comments. Please help me review this again, sir. :pray:

Reply:
Ah, I mean the whole comment... Deleted myself and will merge once CI passes.